### DISCUSSION TITLE
GraphQL deleteEntry mutation

### DISCUSSION THREAD

**Discussion by robinbeatty** [craftcms/cms#14644] [Ideas] [answered]
When using the deleteEntry mutation against an entry in a structure section, is there any way to handle what happens to descendant entries of the deleted entry? e.g. delete with descendants, or unassign the parent ids from all descendants (so immediate children appear at the top level)?

**Comment by brandonkelly** [comment, marked as answer]
All descendants will be deleted along with the targeted entry. You could fetch the descendants beforehand and append them elsewhere using the `save_*` mutation with `appendTo` or `appendToRoot`.
